django/db/models/expressions.py

```diff
@@ -1423,16 +1423,14 @@ class ExpressionList(Func):
 
     template = "%(expressions)s"
 
-    def __init__(self, *expressions, **extra):
-        if not expressions:
-            raise ValueError(
-                "%s requires at least one expression." % self.__class__.__name__
-            )
-        super().__init__(*expressions, **extra)
-
     def __str__(self):
         return self.arg_joiner.join(str(arg) for arg in self.source_expressions)
 
+    def as_sql(self, *args, **kwargs):
+        if not self.source_expressions:
+            return "", ()
+        return super().as_sql(*args, **kwargs)
+
```

Comment:
Refer to https://github.com/django/django/commit/e16d0c176e9b89628cdec5e58c418378c4a2436a#diff-359ab56295cce36b9597e1d65185f695f271955617b40f43df58ce3fe76fd0c8R1275